### ISSUE TITLE
HOMEPAGE html

### ISSUE THREAD

**Issue by houckchris** [houckchris/project2#18]
MAKE HOMEPAGE - A homepage that displays the website logo, and an option of two links: 1) Volunteer or 2) Organization.  For now, the Volunteer link will direct to the volunteer dashboard. And the organization link will direct to the Organization new event form page.  [Future builds or adjustments could route to a more thorough org dashboard, but not now.]

As always: be sure to create new branches before starting new assignments; git push your work to your local branch as often as you normally would save your hw in this fashion; follow the git guide in the group slack for git guidance if you git stuck merging your completed work. I'll approve merges and pull requests ASAP 👍 And please come to me with any questions.
